scripts/snippets_csv/index.js

```diff
@@ -0,0 +1,119 @@
+const ObjectsToCsv = require('objects-to-csv');
+const glob = require('glob-promise');
+const search = require('search-in-file');
+const { gitlogPromise } = require('gitlog');
+const path = require('path');
+const fs = require('fs');
+
+const repoPath = path.join(__dirname, '../../');
+const directories = ['__glossary', '_cci2', '_cci2_ja'];
+
+const log = (message) => {
+  // eslint-disable-next-line no-console
+  console.log('=>', message);
+};
```

Comment:
is the log function here just to let us know what stage we are currently at in retrieving the snippets and when it is completed?

Reply:
Indeed! 
<img width="433" alt="Screen Shot 2021-11-19 at 4 12 48 PM" src="https://user-images.githubusercontent.com/1449325/142706340-446ad426-0cab-4925-9cf9-2caa5f98d1a6.png">